ui/pages/bridge/prepare/prepare-bridge-page.tsx

```diff
@@ -71,6 +75,35 @@ const PrepareBridgePage = () => {
 
   const [rotateSwitchTokens, setRotateSwitchTokens] = useState(false);
 
+  const quoteParams = useMemo(
+    () => ({
+      srcTokenAddress: fromToken?.address,
+      destTokenAddress: toToken?.address || undefined,
+      srcTokenAmount:
+        fromAmount && fromAmount !== '' && fromToken?.decimals
+          ? Numeric.from(fromAmount, 10)
+              .shiftedBy(-1 * Number(fromToken.decimals))
+              .toString()
```

Comment:
Just to clarify, is this an atomic amount or a decimal amount?

Reply:
Ah it's an atomic amount. I feel like we need some helper functions to improve readability of this like

`toAtomicAmount(decimalAmount, decimals)`
`toDecimalAmount(atomicAmount, decimals)` 

or something

Reply:
Or maybe they already exist in the codebase

Reply:
I've been using the `calcTokenAmount` to convert to decimal amounts and will use `calcTokenValue` here